test/jaxp/javax/xml/jaxp/unittest/common/prettyprint/PrettyPrintTest.java

```diff
@@ -180,6 +178,53 @@
         };
     }
 
+    private final String xml1 = "<?xml version=\"1.0\" encoding=\"UTF-8\"?>"
+            + "<a><b>element b</b><c>element c</c><d xml:space=\"preserve\">TRUE</d><e>test</e></a>";
+    private final String expected1 = "<a>\n"
+            + "    <b>element b</b>\n"
+            + "    <c>element c</c>\n"
+            + "    <d xml:space=\"preserve\">TRUE</d>\n"
+            + "    <e>test</e>\n"
+            + "</a>\n";
+
+    private final String xml2 = "<?xml version=\"1.0\" encoding=\"UTF-8\"?>"
+            + "<l0><l1><l2 xml:space=\"preserve\">level 2</l2> level 1 <l2>level 2</l2></l1>"
+            + "<l1 xml:space=\"preserve\"><l2>level 2</l2> level 1 <l2>level 2</l2></l1></l0>";
+    private final String expected2 = "<l0>\n"
+            + "    <l1>\n"
+            + "        <l2 xml:space=\"preserve\">level 2</l2>\n"
+            + "         level 1 \n"
+            + "        <l2>level 2</l2>\n"
+            + "    </l1>\n"
+            + "    <l1 xml:space=\"preserve\"><l2>level 2</l2> level 1 <l2>level 2</l2></l1>\n"
+            + "</l0>\n";
+
+    /*
+     * Bug: 8265073
+     * source and expected output
+     */
+    @DataProvider(name = "preserveSpace")
```

Comment:
You would not need the name, since the default name is the DataProvider method name.